frontend/src/assets/css/variables.scss

```diff
@@ -1,4 +1,25 @@
 :root {
+    // Error colors
+    --error-main: #ff2121;
+
+    // Action colors
+    --action-hover: #2e61ae;
+
+    // Background colors
+    --background-primary: #f7f8fc;
+    --background-secondary: #274f8d;
```

Comment:
This color in figma is used mainly to highlight buttons and labels.
You can call it ( --accent-primary: #274f8d; )

![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/42421305/9d0e3bf7-80a1-4d4f-ba72-795fd20eb255)
![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/42421305/8630d866-d42e-4977-9dbc-396592d9f9d9)

Reply:
I think it`s just a background for different components and we don`t need to give it a unique name. Will see :)